bears/python/PycodestyleBear.py

```diff
@@ -21,9 +27,14 @@ class PycodestyleBear:
     LICENSE = 'AGPL-3.0'
     CAN_DETECT = {'Formatting'}
 
-    @staticmethod
+    @map_setting_to_aspect(
+        max_line_length=LineLength.max_line_length,
+    )
     def create_arguments(
-            filename, file, config_file,
```

Comment:
this line does not need to change.

you only need to add a line above with `self,`